openapi.yaml

```diff
@@ -203,6 +203,22 @@ components:
       title: RoomConfig
       type: object
       x-struct: Elixir.JellyfishWeb.ApiSpec.Room.Config
+    RoomCreateDetailsResponse:
+      description: Response containing room details
+      properties:
+        data:
+          properties:
+            room: 
+              $ref: '#/components/schemas/Room'
+            jellyfish_address:
+              description: Jellyfish instance address
```

Comment:
```suggestion
              description: Jellyfish instance address where the room was created. This might be different than the address of Jellyfish where the request was sent only when running a cluster of Jellyfishes. 
```